src/java.management/share/classes/com/sun/jmx/mbeanserver/ClassLoaderRepositorySupport.java

```diff
@@ -86,8 +86,7 @@ private static class LoaderEntry {
      * loader has been added.
      **/
     private synchronized boolean add(ObjectName name, ClassLoader cl) {
-        List<LoaderEntry> l =
-            new ArrayList<LoaderEntry>(Arrays.asList(loaders));
+        List<LoaderEntry> l = new ArrayList<>(Arrays.asList(loaders));
```

Comment:
I found it strange that ArrayList is used to just append one element to array. I think it's better to just use Arrays.copyOf with length+1.